src/widgets/menu/defaultTemplates.tsx

```diff
@@ -7,17 +7,43 @@ import { cx } from '../../lib/utils';
 import type { MenuComponentTemplates } from './menu';
 
 const defaultTemplates: MenuComponentTemplates = {
-  item({ cssClasses, url, label, count }) {
+  item({
+    cssClasses,
+    url,
+    count,
+    highlighted,
+    label,
+    isFromSearch,
+    searchable,
+  }) {
     return (
       <a className={cx(cssClasses.link)} href={url}>
-        <span className={cx(cssClasses.label)}>{label}</span>
+        {/* @MAJOR: use the `<Highlight>` component instead */}
```

Comment:
```suggestion
        {/* @TODO: use the `<Highlight>` component instead */}
```

no big deal, but this can be done without breaking change